feature/results/src/main/java/com/android/developers/androidify/customize/CustomizeState.kt

```diff
@@ -35,6 +38,11 @@ data class CustomizeExportState(
     ),
     val exportImageCanvas: ExportImageCanvas = ExportImageCanvas(),
     val showImageEditProgress: Boolean = false,
+    val connectedDevice: ConnectedDevice? = null,
+    val watchFaceInstallationStatus: WatchFaceInstallationStatus = WatchFaceInstallationStatus.NotStarted,
+    val watchFaces: List<WatchFaceAsset> = emptyList(),
+    val selectedWatchFace: WatchFaceAsset? = null,
+    val isLoadingWatchFaces: Boolean = false,
```

Comment:
Wdyt about a `WatchFaceSelectionState()` that encapsulates these into one state within `CustomizeExportState()`

Reply:
Great idea! Done